lib/deprecation_toolkit/behaviors/raise.rb

```diff
@@ -37,10 +44,17 @@ class DeprecationRemoved < DeprecationException
       def initialize(current_deprecations, recorded_deprecations)
         removed_deprecations = recorded_deprecations - current_deprecations
 
+        record_message =
+          if defined?(RSpec)
```

Comment:
Instead of checking if the RSpec constant is defined I'd prefer to look which test runner we are currently using.

Maybe we could do something like

```ruby
Deprecation_toolkit.test_runner == 'minitest'
```

You should be able to set it when the minitest plugin loads https://github.com/Shopify/deprecation_toolkit/blob/b351b055ae474d64fd427829237276568c432547/lib/minitest/deprecation_toolkit_plugin.rb#L18

Reply:
👍